db/migrate/20211105044524_update_patients_latest_assessment_at_by_assessment_reported_at.rb

```diff
@@ -0,0 +1,49 @@
+class UpdatePatientsLatestAssessmentAtByAssessmentReportedAt < ActiveRecord::Migration[6.1]
+  def change
+    reversible do |dir|
+      dir.up { populate_latest_assessment_fields(:reported_at) }
+      dir.down { populate_latest_assessment_fields(:created_at) }
+    end
+  end
+
+  def populate_latest_assessment_fields(field)
+    # reset :latest_assessment_at and :latest_assessment_symptomatic to NULL
+    execute <<-SQL.squish
+      UPDATE patients
+      SET patients.latest_assessment_at = NULL, patients.latest_assessment_symptomatic = NULL
+      WHERE patients.purged = FALSE
+    SQL
```

Comment:
Normally I'd probably not care, but I want to make this migration as low risk as possible, given that we are incorporating it into an emergency patch. So in the spirit of that, why do we need to first null out `latest_assessment_at` and `latest_assessment_symptomatic`, won't they be set by the next two commands?

Reply:
they will be set on the next two commands but if the current value is set to something other than `nil` and the value it's supposed to change to is `nil`, it won't get changed because these commands only set the values for patients that do have a latest assessment. there probably won't be many patients with this case but just wanted to make sure the data is 100% accurate